src/astro/package.json

```diff
@@ -11,15 +11,10 @@
     "build-css": "tailwindcss build -i src/styles/styles.css -o public/styles/styles.css"
   },
   "dependencies": {
-    "@astrojs/mdx": "^1.1.1",
-    "@astrojs/react": "^3.0.3",
+    "@astrojs/mdx": "^1.0.3",
     "@astrojs/rss": "^3.0.0",
-    "@astrojs/tailwind": "^5.0.2",
-    "@types/react": "^18.2.28",
-    "@types/react-dom": "^18.2.13",
-    "astro": "^3.3.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "@astrojs/tailwind": "^5.0.0",
+    "astro": "^3.2.4",
```

Comment:
Same for astro and tailwind.

Reply:
Good catch. Corrected this in the next commit. See above.